src/widgets/menu-select/menu-select.js

```diff
@@ -96,8 +90,8 @@ export default function menuSelect({
   templates = defaultTemplates,
   transformItems,
 }) {
-  if (!container || !attribute) {
```

Comment:
This is checked in the connector (see https://github.com/algolia/instantsearch.js/pull/3492#discussion_r254757352).